pkg/process/checks/net.go

```diff
@@ -117,7 +124,7 @@ func (c *ConnectionsCheck) IsEnabled() bool {
 	}
 
 	_, npmModuleEnabled := c.syscfg.EnabledModules[sysconfig.NetworkTracerModule]
-	return npmModuleEnabled && c.syscfg.Enabled
+	return npmModuleEnabled && c.syscfg.Enabled && flavor.GetFlavor() == flavor.ProcessAgent
```

Comment:
[nit] Maybe have this as an if statement in after the darwin check so we can document the behavior with a comment in a similar fashion.

```go
// Comment about this behavior
if flavor.GetFlavor() != flavor.ProcessAgent {
    return false
}
```

Reply:
Done